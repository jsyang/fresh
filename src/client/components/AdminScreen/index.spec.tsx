import {h} from 'preact';
import {render} from 'preact-render-to-string';

import {AdminScreen} from '.';

describe('components/AdminScreen', () => {
    it('should match loading snapshot', () => {
        expect(render(
            <AdminScreen
                rooms={[]}
                tasks={[]}
            />
        )).toMatchSnapshot();
    });
});
