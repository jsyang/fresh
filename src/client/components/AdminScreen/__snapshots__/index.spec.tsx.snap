// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/AdminScreen should match loading snapshot 1`] = `"Loading rooms and tasks..."`;
